docs/index.md

```diff
@@ -0,0 +1,312 @@
+
+<img src="/docs/images/suggest.png" width="300"  />
+
+```java
+// Attach a listener to the locator task since the LocatorTask may or may not be loaded the
+// the very first time a user types text into the search box.  If the Locator is already loaded,
+// the following listener is invoked immediately.
+mLocator.addDoneLoadingListener(new Runnable() {
+    @Override
+    public void run() {
+        final ListenableFuture> suggestionsFuture = mLocator.suggestAsync(query, suggestParams);
+        // Attach a done listener that executes upon completion of the async call
+        suggestionsFuture.addDoneListener(new Runnable() {
+            @Override
+            public void run() {
+                try {
+                    // Get the suggestions returned from the locator task.
+                    // Process and display
+
+                } catch (Exception e) {
+                   dealWithException(e);
+            }
+          }
+        });
+    }
+});
+// Initiate the asynchronous call
+mLocator.loadAsync();
+```
+
+## Geocoding
+
+Once a suggestion in the list has been selected by the user, the suggested address is geocoded using the `geocodeAsync` method of the `LocatorTask`. Along with the address, specific [geocoding parameters](https://developers.arcgis.com/android/latest/guide/search-for-places-geocoding-.htm#ESRI_SECTION2_48C5C281B21B4BF1BBBDBCEA71F105B9) can be set to tune the results. For example, in the Maps App, we set the preferred location and refine that further by setting a boundary of the area to search for matching addresses.
+
+```java
+mGeocodeParams = new GeocodeParameters();
+// Set max results and spatial reference
+mGeocodeParams.setMaxResults(2);
+mGeocodeParams.setOutputSpatialReference(mMapView.getSpatialReference());
+// Use the centre of the current map extent as the location
+mGeocodeParams.setSearchArea(calculateSearchArea());
+mGeocodeParams.setPreferredSearchLocation(mLocation);
+```
+
+```java
+<ListenableFuture> locFuture = mLocator.geocodeAsync(matchedAddress, mGeocodeParams);
+// Attach a done listener that executes upon completion of the async call
+locFuture.addDoneListener(new Runnable() {
+    @Override
+        public void run() {
+          try {
+            List locationResults = locFuture.get();
+          showSuggestedPlace(locationResults, address);
+          } catch (Exception e) {
+            dealWithException(e);
+       }
+    }
+});
+```
+
+## Reverse geocoding
+
+The Map App uses the built-in map magnifier to help users fine tune a location on the map for reverse geocoding. The magnifier appears after a long-press on the map view. Once the long-press is released, the map point is reverse geocoded.
+
+On long press                    | Reverse geocode result
+:-------------------------------:|:-------------------------------------:
+<img src="/docs/images/reverse_geocode.png" width="300"  />|<img src="/doc/images/reverse_geocode2.png" width="300"  />
```

Comment:
The second image appears broken.

Reply:
Nice catch here. I stared at that relative path for entirely too long to figure out what the issue was...